local/install_deps_linux.bash

```diff
@@ -98,7 +98,7 @@ else
 
   export CLOUD_SDK_REPO="cloud-sdk"
   export APT_FILE=/etc/apt/sources.list.d/google-cloud-sdk.list
-  export APT_LINE="deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main"
+  export APT_LINE="deb https://packages.cloud.google.com/apt $CLOUD_SDK_REPO main"
```

Comment:
trying to install cloud-sdk now fails on ubuntu20 unless using https.

https usage is also how they do it in https://cloud.google.com/sdk/docs/install